cmd/download.go

```diff
@@ -55,15 +89,42 @@ var downloadCmd = &cobra.Command{
 			return fmt.Errorf("Invalid release specified. X.Y.Z format expected: %s", release)
 		}
 
-		if !versionRE.MatchString(hubVersion) {
-			return fmt.Errorf("Invalid hub-version specified. X.Y.Z format expected: %s", hubVersion)
+		// Either hubVersion or mceVersion must be set
+		if (hubVersion != "" && acmVersion != "") || (hubVersion != "" && mceVersion != "") {
```

Comment:
Bump cobra to 1.5.0 then we can just use something like: MarkFlagsMutuallyExclusive

https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive